service/src/main/java/com/myhome/services/springdatajpa/MailSDJpaService.java

```diff
@@ -0,0 +1,49 @@
+package com.myhome.services.springdatajpa;
+
+import com.myhome.domain.User;
+import com.myhome.services.MailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Profile;
+import org.springframework.mail.MailSendException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+@ConditionalOnProperty(value = "spring.mail.debug", havingValue = "false", matchIfMissing = false)
+@RequiredArgsConstructor
+public class MailSDJpaService implements MailService {
+
+  private final JavaMailSender mailSender;
+
+  @Value("${spring.mail.username}")
+  private String username;
+
+  private void send(String emailTo, String subject, String message) throws MailSendException {
+    SimpleMailMessage mailMessage = new SimpleMailMessage();
+    mailMessage.setFrom(username);
+    mailMessage.setTo(emailTo);
+    mailMessage.setSubject(subject);
+    mailMessage.setText(message);
+    mailSender.send(mailMessage);
```

Comment:
Send can throw exception, maybe it would be good to wrap it with try catch (and log and return false in case of failure - after review changes this method will return boolean) ?